src/Http/Middleware/Authenticate.php

```diff
@@ -10,10 +10,11 @@ class Authenticate extends Middleware
      * Get the path the user should be redirected to when they are not authenticated.
      *
      * @param  \Illuminate\Http\Request  $request
+     * @param  array  $guards
      *
      * @return string
      */
-    protected function redirectTo($request)
+    protected function redirectTo($request, $guards)
```

Comment:
I personally think it should be type-hinted to `array $guard` since `handle()` method uses that. However it need to happen on Laravel Framework code as well.

Reply:
@crynobone you're right. Let's do it correct from the first go.

Reply:
Sent in a PR for this.